### ISSUE TITLE
LDN 2.3 - Smash Ultimate - Graphical Glitch on Stage Selection Screen

### ISSUE THREAD

**Issue by cow-killer** [Ryujinx/Ryujinx#2350]
Hardware:
-Arch Linux
-GTX 1660 (driver 465.31)
-i5-8400
-24 GB RAM

When I join a lobby through WLAN, or I create it myself, this is what the stage selection screen looks like:

![Smash SSS graphical bug](https://i.imgur.com/Tntvmhu.png)

After picking a stage the emulator crashes. Here's part of the log:
```
00:02:11.574 |I| HLE.OsThread.10 ServiceNv Create: Created map 1795 with size 0x0003a000!
00:02:11.574 |I| HLE.OsThread.10 ServiceNv Create: Created map 1796 with size 0x00040000!
00:02:11.575 |I| HLE.OsThread.10 ServiceNv Create: Created map 1797 with size 0x00040000!
00:02:11.576 |I| HLE.OsThread.10 ServiceNv DecrementMapRefCount: Deleted map 1719!
*** stack smashing detected ***: terminated
./ryujinx.sh: line 3: 1288086 Aborted                 (core dumped) MANGOHUD_DLSYM=1 MANGOHUD_CONFIG=gl_bind_framebuffer=0 mangohud __GL_THREADED_OPTIMIZATIONS=1 __GL_SYNC_TO_VBLANK=0 gamemoderun ryujinx_ldn2.3/./Ryujinx
```

I'm guessing I need to disable threaded optimization/gamemode? And sorry if you aren't taking LDN bugs at the moment; I can remove this if you want.

Edit: mangohud, threaded optimization and gamemode all disabled, tried again, same result.

**Comment by jduncanator** [comment]
Do you have a full log you could upload?

**Comment by cow-killer** [comment]
[Ryujinx_1.0.0-ldn2.3_2021-06-07_09-50-23.log](https://github.com/Ryujinx/Ryujinx/files/6609518/Ryujinx_1.0.0-ldn2.3_2021-06-07_09-50-23.log)

**Comment by Shahil-Ayato** [comment]
I haven’t been able to reproduce this on 3 different systems at all anymore, I used to be able to on the old LDN builds but not anymore, if someone can please share. Otherwise I think this should be closed

**Comment by cow-killer** [comment]
This was fixed with 2.4, I'll close this.
